lib/rules/no-control-regex.js

```diff
@@ -51,29 +51,47 @@ module.exports = {
             return null;
         }
 
+
+        const controlChar = /[\x00-\x1f]/g; // eslint-disable-line no-control-regex
+        const multipleSlashes = /\\+/g;
+        const multipleSlashesAtEnd = /\\+$/g;
```

Comment:
Technically these both match one or more backslashes, right? Or is the goal to match 2 or more?